sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 12.0.2 (Unreleased)
+
+- Improvements to sample code.
```

Comment:
Sample code doesnt get shipped as part of the package, so we can skip this.

Reply:
Removed. Was thinking that this might intrigue customers to check out updated samples.